projects/graph/src/graph_demo.py

```diff
@@ -66,8 +66,8 @@ def main(style, numNodes, numEdges):
 
 if __name__ == '__main__':
     style = "default"
-    numNodes = 5
-    numEdges = 5
+    numNodes = random.randint(3,7)
+    numEdges = random.randint(3,10)
 
```

Comment:
This works to make the graph display differently each time. However, since the vertices and edges are hard coded in, there are always the same number of them, connected in the same way - the glyphs just appear in different places.